pages/resources/guild.mdx

```diff
@@ -922,6 +922,7 @@ Creates a new guild. Returns a [guild](#guild-object) object on success. Fires a
 | channels? ^2^                  | array[partial [channel](/resources/channel#channel-object) object] | Channels in the new guild                                                                                                  |
 | afk_channel_id?                | ?snowflake                                                         | The ID of the guild's AFK channel; this is where members in voice idle for longer than `afk_timeout` are moved             |
 | afk_timeout?                   | integer                                                            | The AFK timeout of the guild (one of 60, 300, 900, 1800, 3600, in seconds)                                                 |
+| staff_only? ^3^                | boolean                                                            | If new guild available for staff only.                                                                                     |
```

Comment:
```suggestion
| staff_only? ^3^                | boolean                                                            | Whether the new guild will only be accessible for Discord employees                                                                                     |
```

Reply:
Maybe also add an "(adds the [`INTERNAL_EMPLOYEE_ONLY` guild feature](#guild-feature))" at the end